crates/host/src/wasmbus/mod.rs

```diff
@@ -1903,6 +1913,10 @@ impl Host {
             ("hostcore.os".into(), OS.into()),
             ("hostcore.osfamily".into(), FAMILY.into()),
         ]);
+
+        let meter = global::meter("wasmcloud-host");
```

Comment:
Should we make this a const we also reference in main when we call `configure_tracing`?

Reply:
I've made this use `CARGO_PKG_NAME` inside of the host crate, we can decide to revisit this and turn it into a constant or leave as is if it doesn't feel that important.